.circleci/config.yml

```diff
@@ -277,6 +277,8 @@ jobs:
   truffle-sample-project:
     docker:
       - image: circleci/node:12
+    # npm install runs out of memory on medium when installing Truffle's dependencies
```

Comment:
Wtf... how much memory is that :-D? Kind of crazy :-).

Reply:
Truffle's build process is actually very heavy for a JS app. Typescript compilation takes several minutes and that's probably where the memory goes. I don't really want to go too deep into it so for now I'm just increasing the container size.

But I'm also switching to yarn right now since the dependency injection does not work well with npm for Truffle. Maybe that eats less, though if it's because of TS then probably not.

I already had to deal with that a bit when I was creating this config for the first time. At least in terms of build time (did not have any memory issues on `medium` back then). The CI commands are structured in such a way as to avoid rerunning `yarn install` for Truffle if not necessary because that rebuilds everything.